<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoneSegmentationWidget</class>
 <widget class="QWidget" name="BoneSegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoneSegmentationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QToolButton" name="toolButtonShowVoi">
     <property name="text">
      <string>Show Voi</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonSetVoi">
     <property name="text">
      <string>Set Voi</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUnsharpMaskAmount">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Sigma:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUnsharpMaskSigma">
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUnsharpMaskThreshold"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonUnsharpMask">
       <property name="text">
        <string>UnSharp Mask</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Number of Thresholds:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBodyMaskNumberOfThresholds">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBodyMaskThreshold">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonBodyMask">
       <property name="text">
        <string>Body Mask</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Slice By Slice</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="checkBoxBinaryMorphologicalClosing">
          <property name="text">
           <string>Binary Morphological Closing</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Structuring Element Radisu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxBinaryMorpholocialClosingRadius">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Fill Holes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButtonNone">
             <property name="text">
              <string>None</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupFillHoles</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonBinaryFillHole">
             <property name="text">
              <string>Binary Fill Hole</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupFillHoles</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonVotingBinaryHoleFilling">
             <property name="text">
              <string>Voting Binary Hole Filling</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupFillHoles</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonVotingBinaryIterativeHoleFilling">
             <property name="text">
              <string>Voting Binary Iterative Hole Filling</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupFillHoles</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Voting Binary Hole Filling</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Radius:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxVotingBinaryHoleFillingRadius">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Iterative:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxVotingBinaryHoleFillingIterative">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxBinaryMorphologicalOpening">
          <property name="text">
           <string>Binary Morphological Opening</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Structuring Element Radius:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxStructuringElementRadius">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButtonOtsuThresholdSliceBySlice">
          <property name="text">
           <string>Otsu Threshold Slice By Slice</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarOtsuThresholdSliceBySlice">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Number of Components:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxNumberOfComponents">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSliceBySliceThresholdLower">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSliceBySliceThresholdUpper">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonThreshold">
       <property name="text">
        <string>Threshold</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QToolButton" name="toolButtonPaintBrush">
       <property name="text">
        <string>Paint Brush</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPaintBrushSize">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonImageToSurface">
     <property name="text">
      <string>Image To Surface</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonSaveSegmentation">
     <property name="text">
      <string>Save Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonLoadSegmentation">
     <property name="text">
      <string>Load Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonRedo">
     <property name="text">
      <string>Redo</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>840</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupFillHoles"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
